spec/controllers/quotes_controller_spec.rb

```diff
@@ -0,0 +1,71 @@
+require 'rails_helper'
+
+RSpec.describe QuotesController, type: :controller do
+  let(:quote) { create(:quote) }
+
+  describe "#new" do
+    subject { get :new }
+    it "renders the new template" do
+      expect(subject).to render_template(:new)
+    end
+
+    it "instantiates a new Quote variable" do
+      subject
+      expect(assigns(:quote)).to be_a_new Quote
+    end
+  end
+
+  describe "#create" do
+    context "with valid parameters" do
+      subject { post :create, quote: attributes_for(:quote) }
+      it "changes the quotes count by +1" do
+        expect { subject }.to change { Quote.count }.by(1)
+      end
+
+      it "redirects to the show page" do
+        expect(subject).to redirect_to quote_path(Quote.last)
+      end
+    end
+    context "with invalid parameters" do
+      subject { post :create, quote: attributes_for(:quote).merge(body: nil) }
```

Comment:
`quote: attributes_for(:quote).merge(body: nil)`
is the same as
`quote: attributes_for(:quote, body: nil)`